src/Cardano/Wallet/CoinSelection/Random.hs

```diff
@@ -202,29 +200,20 @@ processTxOut maxNumInputs input txout =
                         val2 - val1
                     else
                         val1 - val2
-                targetRange = (mkTargetRange . coin) txout
+                targetRange = mkTargetRange txout
 
         selectedBalance
             :: [(TxIn, TxOut)]
             -> Coin
         selectedBalance = Coin . sum . (map (getCoin . coin . snd))
 
-mkTargetRange :: Coin -> TargetRange
-mkTargetRange val =
-    fromMaybe (privacyOffTargetRange val) (tryCanonicalTargetRange val)
-    where
-        tryCanonicalTargetRange :: Coin -> Maybe TargetRange
-        tryCanonicalTargetRange coin@(Coin v) = do
-            let targetMin = coin
-            targetAim <-
-                if isValidCoin (Coin $ 2*v) then Just (Coin $ 2*v) else Nothing
-            targetMax <-
-                if isValidCoin (Coin $ 3*v) then Just (Coin $ 3*v) else Nothing
```

Comment:
we are not afraid of situation that for instance targetMax will exceed upperBound of what Coin can have???